packages/taiko-client/driver/anchor_tx_constructor/anchor_tx_constructor.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/decred/dcrd/dcrec/secp256k1/v4"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/consensus/taiko"
```

Comment:
```suggestion
	consensus "github.com/ethereum/go-ethereum/consensus/taiko"
```
Lets use `consensus` as the package name instead of `taiko`